### ISSUE TITLE
[Files app] Clicking both "Done" buttons at folder creation breaks the Files app

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/ios-app#221]
### Steps to reproduce
1. Perform steps from #203 in iOS Files app
2. Tap the "New folder" button and enter name "Test"
3. Tap bottom-right "Done" button ==> Keyboard disappears, but no other immediate response
4. Tap bottom-top "Done" button (don't wait too long) ==> Error message "The name "Test" is already taken. Please choose a different name" appears and disappears automatically, then the Files app is stuck. Top-left "Cancel" button doesn't work. Only force-quitting the app helps.

Step 2. | Step 3. | Step 4.
:--: | :--: | :--:
![image](https://user-images.githubusercontent.com/214010/51211852-03311000-1917-11e9-8492-e03baa1b9cc1.png) | ![image](https://user-images.githubusercontent.com/214010/51211891-1a6ffd80-1917-11e9-86b5-10ccd7ffc5f1.png) | ![image](https://user-images.githubusercontent.com/214010/51211912-28be1980-1917-11e9-9a0b-73678bbf5f2d.png)
**Stuck** | **Force-quit** | **Network Link Conditioner**
![image](https://user-images.githubusercontent.com/214010/51211891-1a6ffd80-1917-11e9-86b5-10ccd7ffc5f1.png) | ![image](https://user-images.githubusercontent.com/214010/51211960-45f2e800-1917-11e9-8597-a80fd5a866fc.png) |![image](https://user-images.githubusercontent.com/214010/51211826-f3b1c700-1916-11e9-808b-edb818960c6c.png)

Using Network Link Conditioner might help to simulate slow network.

In other tests I was a little slower, then a zombie folder remained. Sadly, I never captures this with logging enabled.

### Expected behaviour
Double-input should be prevented. First action needs to be finished before another action can be started.

### Actual behaviour
Files app is stuck. Top-left "Cancel" button doesn't work. Only force-quitting the app helps.

### Client
**iOS version:**
iOS 12.1.2 (16C101)

**ownCloud app version:**
ownCloud beta version 1.0 build 96 (https://github.com/owncloud/ios-app/commit/c5c74efd5850c2279ee42b057ca0332696da1569, not shown in the app??)

**Device model:**
Model: iPhone XS Max

@felix-schwarz I have app debug logs

**Comment by jesmrec** [comment]
Not reproducible on my side. @michaelstingl do you still reproduce it?

**Comment by michaelstingl** [comment]
> @michaelstingl do you still reproduce it?

Yes, but I don't have the latest-latest version. I'll try again with the nextTestFlight and provide feedback.

**Comment by michaelstingl** [comment]
Yepp, I still can reproduce with ownCloud beta version 1.0 build 103 (https://github.com/owncloud/ios-app/commit/373776c5dbf0b508e9b369972b7601b0409f0049)
